chain/actors/manifest.go

```diff
@@ -31,9 +31,25 @@ const (
 	RewardKey   = "reward"
 	SystemKey   = "system"
 	VerifregKey = "verifiedregistry"
+	DatacapKey  = "datacap"
 )
 
-func GetBuiltinActorsKeys() []string {
+func GetBuiltinActorsKeys(av actorstypes.Version) []string {
+	if av <= 8 {
```

Comment:
I'd rewrite this to something like 

```
baseActors := []string{
			AccountKey,
			CronKey,
			InitKey,
			MarketKey,
			MinerKey,
			MultisigKey,
			PaychKey,
			PowerKey,
			RewardKey,
			SystemKey,
			VerifregKey,
		}
```

And then return append(baseActors, DatacapKey) if >= 9